aws_lambda_builders/workflows/java_gradle/gradle_resolver.py

```diff
@@ -0,0 +1,30 @@
+"""
+Gradle executable resolution
+"""
+
+import os
+from aws_lambda_builders.path_resolver import PathResolver
+from .utils import OSUtils
+
+
+class GradleResolver(object):
+    def __init__(self, source_dir, os_utils, path_resolver=None):
+        self.source_dir = source_dir
+        self.os_utils = OSUtils() if not os_utils else os_utils
+        self.binary = 'gradle'
+        self.executables = [self.binary]
+        self.path_resolver = PathResolver(binary='gradle', runtime=None) if not path_resolver else path_resolver
+
+    @property
+    def exec_paths(self):
+        if self._includes_gradlew():
```

Comment:
You can simplify this more by simply using `PathResolver('gradlew') or PathResolver('gradle')` to get paths for both :)

Reply:
I don't believe this will work right now for `gradlew` case though right? `gradlew` is included in the project root, so it won't be on the `PATH`.

Reply:
Incorporated #78